akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PrimitiveStateSpec.scala

```diff
@@ -54,6 +54,8 @@ class PrimitiveStateSpec extends ScalaTestWithActorTestKit(PrimitiveStateSpec.co
       probe.expectMessage("eventHandler:1:2")
 
       ref1 ! -1
+      probe.expectTerminated(ref1)
```

Comment:
The failure was
```
java.lang.AssertionError: assertion failed: expected eventHandler:1:2, found onRecoveryCompleted:1
```

The event handler is actually invoked before persisting so next actor is started before 2 has been persisted and therefore only replays 1 and then `onRecoveryCompleted:1`.

Good!

Reply:
Yup :+1: